x-pack/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/examples_list/examples_list.tsx

```diff
@@ -32,19 +37,17 @@ export const ExamplesList: FC<Props> = ({ examples }) => {
 
   return (
     <div data-test-subj="mlFieldDataCardExamplesList">
-      <EuiTitle size="xxxs" className="mlFieldDataCard__valuesTitle">
-        <span>
-          <FormattedMessage
-            id="xpack.ml.fieldDataCard.cardText.examplesTitle"
-            defaultMessage="{numExamples, plural, one {value} other {examples}}"
-            values={{
-              numExamples: examples.length,
-            }}
-          />
-        </span>
-      </EuiTitle>
+      <ExpandedRowFieldHeader>
+        <FormattedMessage
+          id="xpack.ml.fieldDataCard.cardText.examplesTitle"
```

Comment:
Nit - maybe don't need `cardText` in this id - `xpack.ml.fieldDataCard.examplesList.title` ?

Reply:
Renamed here https://github.com/elastic/kibana/pull/85726/commits/13fa2c8f86d766e6db81a72638ed3dd8f968cedb